packages/lambda/src/repositories/nonprofits.js

```diff
@@ -510,6 +516,28 @@ NonprofitsRepository.prototype.upsert = function (model, data) {
 				'status': typeof data.status !== "undefined" ? data.status : model.status,
 				'taxId': typeof data.taxId !== "undefined" ? data.taxId : model.taxId,
 				'receiveDonationNotifications': typeof data.receiveDonationNotifications !== "undefined" ? data.receiveDonationNotifications : model.receiveDonationNotifications,
+			};
+
+			return allModels.Nonprofit.upsert(params).then(async response => {
+				const updatedModel = response[0];
+
+				if (! data.NonprofitAgreements) {
+					return updatedModel;
+				}
+
+				// set the agreements
+
+				// data.NonprofitAgreements: [{ agreementId: 10, isChecked: true }, { agreementId: 11, isChecked: false }];
```

Comment:
should this still be there?

Reply:
Yeah, its meant to be a code comment documenting the shape of the `data.NonprofitAgreements` coming in.